WorkloadManagementSystem/Utilities/ParametricJob.py

```diff
@@ -0,0 +1,164 @@
+""" Utilities to process parametric job definitions and generate
```

Comment:
Also, there's not a single log message in this module. I'd say this deserves at least some debug ones.